README.md

```diff
@@ -56,61 +57,97 @@ const converted = tabletojson.convert(html);
 console.log(converted);
 ```
 
-### Duplicate column headings 
+### Duplicate column headings
 
-If there are duplicate column headings, subsequent headings are suffixed with a count:
+If there are duplicate column headings, subsequent headings are suffixed with a
+count:
 
-```
-// Table
-| PLACE | VALUE | PLACE | VALUE |
-|   abc |     1 |   def |     2 |
+PLACE | VALUE | PLACE | VALUE
+------|-------|-------|------
+  abc |     1 |   def |     2
 
-// Example output
+```js
 [{
   PLACE: 'abc', VALUE: '1',
   PLACE_2: 'def', VALUE_2: '2',
 }]
 ```
 
-### Tables with rowspan 
-
-Having tables with rowspan, the content of the spawned cell must be available in the respective object.
+### Tables with rowspan
 
-```
-// Table
-| PARENT | CHILD | AGE |
-|        |   Tom |   3 |
-| Marry  | Steve |  12 |
-|        |   Sue |  15 |
+Having tables with rowspan, the content of the spawned cell must be available in
+the respective object.
 
-// Example output
```

Comment:
The output of the conversion is a JSON object. and not HTML code.

What is this HTML code standing for?

Reply:
It's not the output, but instead the input. It's a real HTML table since markdown don't support rowspan, but instead allow embeded html, so this way we can show it.